src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -303,6 +307,77 @@ export default class RadarTargetModel {
         return [true, 'AMEND ALTITUDE'];
     }
 
+    /**
+     * Generate a string to be used for the first row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowOne
+     * @returns {string}
+     */
+    buildDataBlockRowOne() {
+        let dataBlockRowOne = this.aircraftModel.callsign;
+
+        if (this.aircraftModel.model.weightclass === 'H') {
+            // using empty space here on purpose so this gets rendered
+            // appropriately within a canvas
+            dataBlockRowOne += '  H';
+        }
+
+        return dataBlockRowOne;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwoPrimaryInfo
+     * @returns {string}
+     */
+    buildDataBlockRowTwoPrimaryInfo() {
+        const aircraftAltitude = round(this.aircraftModel.altitude / 100);
+        const aircraftSpeed = round(this.aircraftModel.groundSpeed / 10);
+
+        return `${leftPad(aircraftAltitude, 3)}  ${leftPad(aircraftSpeed, 2)}`;
+    }
+
+    /**
+     * Generate a string to be used for the second row of a datablock
+     * when the timeshare section is active
+     *
+     * @for RadarTargetModel
+     * @method buildDataBlockRowTwoSecondaryInfo
+     * @returns {string}
+     */
+    buildDataBlockRowTwoSecondaryInfo() {
+        const scratchPadText = _pad(this._scratchPadText, 4, ' ');
+        const aircraftModelIcao = _pad(this.aircraftModel.model.icao.toUpperCase(), 4, ' ');
```

Comment:
Super sorry, I messed up-- the scratchpad should be _three_ characters max, not four. I saw 4 characters in the vSTARS documentation, and was like "that doesn't feel right, but it must be right"-- it must have been an old screenshot, because, in action, vSTARS itself also uses three characters, which is definitely correct.

These should both be changed to 3 characters. Perhaps this mistake of mine indicates that there's a point to be made to use an enum that's linked to theme changes instead of hardcoding this value. A todo to that effect would be great :smile: 

---

Also, the use of `_pad()` here is causing the value to be padded from the left, it should always be padded at the end only, in order for the first character to be aligned with the first character of the first line. `_padEnd()` should do what we need here.

![image](https://user-images.githubusercontent.com/5103735/48981881-08a57300-f0a9-11e8-94df-58ec87a2900b.png)

Reply:
<img width="106" alt="screen shot 2018-11-25 at 1 41 26 pm" src="https://user-images.githubusercontent.com/3325985/48983702-de0ee680-f0b7-11e8-9526-5a566184a3d9.png">
<img width="107" alt="screen shot 2018-11-25 at 1 41 30 pm" src="https://user-images.githubusercontent.com/3325985/48983703-de0ee680-f0b7-11e8-8b05-0570d66a0e36.png">

Reply:
@erikquinn just noticed these screenshots are still wrong.  `AAA-BBBB`, I'm missing a character in there.